packages/nodes-base/nodes/FileMaker/FileMaker.node.ts

```diff
@@ -617,13 +609,7 @@ export class FileMaker implements INodeType {
 					value: '',
 				});
 
-				let layouts;
-				try {
-					layouts = await layoutsApiRequest.call(this);
-				} catch (error) {
-					throw new NodeOperationError(this.getNode(), error as Error);
-				}
-				for (const layout of layouts) {
+				for (const layout of await layoutsApiRequest.call(this)) {
```

Comment:
I find `of await` difficult to debug. this would make a lot more sense of the object being looped on was a lazy iterator, as that could help avoid loading all data in memory. but in this case, all data is going to be first loaded in memory, so might as well refer it in the `layouts` variable, to make debugging easier.